reactrails-in-reagent.iml

```diff
@@ -0,0 +1,148 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
Not sure if Idea files should go into github. We had issues with ruby projects, so we gitignore all idea files.